starknet_devnet/devnet_config.py

```diff
@@ -317,9 +327,9 @@ def parse_args(raw_args: List[str]):
         help="Specify the seed for randomness of accounts to be predeployed",
     )
     parser.add_argument(
-        "--hide-predeployed-accounts",
```

Comment:
Perhaps instead of introducing a breaking change, we could keep the `--hide-predeployed-accounts` flag, but just have it print a deprecation notice and notify the user to use the other flag. Or this `parser.add_argument(...` could accept two entry points (`--hide-predeployed-accounts` and `--hide-predeployed-contracts`)